node/flatpak-node-generator.py

```diff
@@ -1317,6 +1384,31 @@ def process_lockfile(self, lockfile: Path) -> Iterator[Package]:
             yield self.parse_package_section(lockfile, section)
 
 
+class YarnRCFileProvider(RCFileProvider):
+    rcfile_name = '.yarnrc'
+
+    @classmethod
+    def parse_rcfile(cls, rcfile: Path) -> Dict[str, str]:
+        parser_re = re.compile(r'^(\S+)\s+(?:"(.+)"|(\S+))', re.MULTILINE)
+        with open(rcfile, 'r') as r:
+            rcfile_text = r.read()
+        result = {}
+        for key, quoted_val, val in parser_re.findall(rcfile_text):
+            result[key] = quoted_val + val
```

Comment:
would probably be clearer as `quoted_val or val` or similar, took me a moment to figure out the concat (also, does `.yarnrc` actually allow multiline strings?)